mcnp_to_csv.py

```diff
@@ -0,0 +1,48 @@
+# 1. generate array of boundaries for x, y, and z
+# 2. generate array of centroid (x,y,z), result, relative error
+import argparse
+import os
+
+def mcnp_to_csv(mcnpFile, outputFile):
+    # append output filename with .csv regardless of what its file extension is
+    output_filename, _ = os.path.splitext(outputFile)
+    output_filename += ".csv"
+
+    # open files
+    with open(mcnpFile, 'r') as mcnpFile, open(output_filename, 'w') as outputFile:
+        # get array of boundaries for x,y,z
+        # 1a. find line that has x boundaries
+        mcnpLine = mcnpFile.readline()
+        while "X direction:" not in mcnpLine:
+            mcnpLine = mcnpFile.readline()
+        # 1b. split line into all the boundaries, write to file with comma separation
+        # do this 3 times for the 3 axes
+        num_dims = 3
+        for dim in range(num_dims):
+            outputStr = ','.join(mcnpLine.strip().split()[2:]) + '\n'
+            outputFile.write(outputStr)
+            mcnpLine = mcnpFile.readline()
```

Comment:
```suggestion
        dims = ['x', 'y', 'z']
        for dim in dims:
            outputStr = ','.join(mcnpLine.strip().split()[2:]) + '\n'
            outputFile.write(outputStr)
            mcnpLine = mcnpFile.readline()
```
Here's a somewhat arbitrary suggestion. It won't change the functionality at all but might make it even clearer what's actually happening in this block (useful when you or someone else returns to this script after some time away).